code/__HELPERS/priority_announce.dm

```diff
@@ -86,34 +116,104 @@
  * message - the message contents of the announcement.
  * title - the title of the announcement, which is often "who sent it".
  * alert - whether this announcement is an alert, or just a notice. Only changes the sound that is played by default.
- * from - who sent the announcement, in the case of communications consoles or the like.
  * html_encode - if TRUE, we will html encode our title and message before sending it, to prevent player input abuse.
- * players - optional, a list mobs to send the announcement to. If unset, sends to all players.
+ * players - optional, a list mobs to send the announcement to. If unset, sends to all palyers.
```

Comment:
Hope you aren't copy-pasting without looking.

Reply:
me when i open PRs while sleep deprived

Reply:
you never actually pushed the commit that fixes this @Absolucy